src/components/Card/FrontSide.tsx

```diff
@@ -19,13 +19,11 @@ type Props = {
 }
 
 const FrontSide: React.FC<Props> = ({ model, cardType, focusedField }) => {
-  const { overrides, translations } = useContext(LibraryContext)
+  const { overrides, translations, requiresName } = useContext(LibraryContext)
   const [numberLayout, setNumberLayout] = useState<LayoutRectangle | null>(null)
   const [nameLayout, setNameLayout] = useState<LayoutRectangle | null>(null)
-  const [
-    expirationLayout,
-    setExpirationLayout,
-  ] = useState<LayoutRectangle | null>(null)
+  const [expirationLayout, setExpirationLayout] =
+    useState<LayoutRectangle | null>(null)
```

Comment:
This change looks like not modified on purpose.

Reply:
Yes, sorry it was not modified on purpose. I guess my editor code prettier did. I'm fixing it now

Reply:
Done!